local-queue.js

```diff
@@ -1,65 +1,42 @@
-const queues = new Map()
-
-
-// Get or create queue
-function getOrCreateQueue(queueName) {
-  if (!queues.has(queueName)) {
-    queues.set(queueName, new Array())
+// A LocalQueue implementation
+class LocalQueue {
+  constructor(queueName) {
+    this.name = queueName
+    this.queue = new Array()
```

Comment:
Keep in mind for the future: also have a Set for quick `has` operations. Furthermore, you might have to make the methods `async` again to be consistent with the Redis queue in the future.

Reply:
created an issue #39